packages/testing/src/render.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { MockedProvider } from '@apollo/react-testing'
+
+import Routes from '~/web/Routes'
+
+const AllTheProviders = ({ mocks, children }) => {
+  return (
+    <MockedProvider mocks={mocks} addTypename={false}>
```

Comment:
I added that part because the Apollo docs said so: https://www.apollographql.com/docs/react/development-testing/testing/

I'm not exactly sure why tho, so if you think it's needed I'm happy to make it overridable!